docs/review_process.md

```diff
@@ -0,0 +1,138 @@
+- writing comments (most of the time, it's a build status, either failure with logs or success)
+- merging pull requests
+- closing issues (side effect of merging pull requests)
+- starting CI builds
+- assigning CI status (running/failed/successful)
+
+## green build
+
+The first important prerequisite is ensuring your PR is green (build is successful).
+It requires a bit of patience, because there are many PRs are running, and we're building a lot of configurations for a numerous versions of libraries.
+Keep attention to the error messages from the bot, and address all the build failures.
+The bot tries to provide all the helpful information needed to understand and reproduce an issue, such as:
+
+- the profile that failed (in other words, the configuration - architecture, operation system, compiler, etc.)
+- failed command line (it might have failed on early stages - e.g. recipe syntax errors, or later stages, like build or test)
+- logs (contains actual output of the build process - not that some important logs like `configure.log` or `CMakeError.log` are not captured, it's just stdout/stderr)
+
+If you struggle to fix build errors yourself, you always may ask for the help, either by tagging individual users, or by adding label [Help Wanted](https://github.com/conan-io/conan-center-index/pulls?q=is%3Aopen+is%3Apr+label%3A%22help+wanted%22).
+
+### Unexpected error
+
+Sometimes, build fails with `Unexpected error` message. This indicates an infrastructure problem, and usually it's unrelated to the changes within PR itself.
+Keep in mind CCI is still in early access (consider it beta), and there are still some instabilities. Especially, as we're using lots of external services,
+which might be inaccessible (GitHub API, docker hub, etc.) and may result in intermittent failures.
+So, what to do once `Unexpected error` was encountered? You may consider re-running the build, and there are several ways to do it:
+
+- do an empty commit: `git commit --allow-empty`
+- update the last commit: `git commit --amend --no-edit && git push -f`
+- close pull request, wait 15 seconds, re-open pull request
+
+Sometimes it's necessary to restart the build several times.
+If an `Unexpected error` persists, you may assign an [Infrastructure](https://github.com/conan-io/conan-center-index/pulls?q=is%3Apr+is%3Aopen+label%3ADocs+label%3Ainfrastructure) label and tag [@jgsogo](https://github.com/jgsogo) and [@danimtb](https://github.com/danimtb) asking for the help with CI.
+Alternatively, just [open a new issue](https://github.com/conan-io/conan-center-index/issues/new/choose).
+
+## avoiding conflicts
+
+It's recommended to rebase your changes on top of the master branch to avoid conflicts.
+Right now, neither GitHub itself nor conan-center-bot notify about merge conflicts, so it's the contributor's responsibility to periodically check for the conflicts.
```

Comment:
GitHub does flag PR with conflicts in master... I would have sworn the bot was blocked by this as well.

![image](https://user-images.githubusercontent.com/16867443/107722336-b4e4cb80-6cac-11eb-8be3-e909d45e6864.png)

Reply:
it blocks the merge, that's right. the problem you never get the notifications about conflicts, so the only way, for now, is to check periodically. in the future, I guess, the bot could assign a label.